### PR TITLE
arkouda-docker: refactor arkouda-udp-server and update README

### LINKED ISSUE #71: arkouda-docker: refactor arkouda-udp-server and update README

### PR DESCRIPTION
**Pull request by hokiegeek2**
Closes #71 

1. refactored arkouda-udp-server Dockerfile to decrease image size
2. updated README

### CODE REVIEW

**Review by Ethan-DeBandi99 — APPROVED** [review]
Looks good!

File: arkouda-docker/README.md [hunk 1]
```diff
@@ -8,10 +8,10 @@ The arkouda-full-stack image starts a one-locale arkouda\_server and provides an
...
 ```
 # set env variables
-export CHAPEL_SMP_IMAGE=chapel/chapel-gasnet-smp:1.28.0
-export ARKOUDA_BRANCH_NAME=2022.11.17
-export ARKOUDA_DISTRO_NAME=v2022.11.17
-export ARKOUDA_DOWNLOAD_URL=https://github.com/Bears-R-Us/arkouda/archive/refs/tags/v2022.11.17.zip
+export CHAPEL_SMP_IMAGE=chapel/chapel-gasnet-smp:1.29.0
+export ARKOUDA_BRANCH_NAME=2023.01.11
+export ARKOUDA_DISTRO_NAME=v2023.01.11
+export ARKOUDA_DOWNLOAD_URL=https://github.com/Bears-R-Us/arkouda/archive/refs/tags/v2023.01.11.zip
```
Comment (pierce314159) [inline arkouda-docker/README.md:14]: is there a reason we're not updating to the `v2023.02.08` tag? Is it that we don't have a container for that version yet?

**Review by pierce314159 — APPROVED** [review]
looks good to me! just one question about the arkouda version used

File: arkouda-docker/README.md (hunk 1 above)
Comment (hokiegeek2) [reply arkouda-docker/README.md:14]: @pierce314159 will do that tag next week 💯

**Pull request merged**